Text.txt

```diff
@@ -0,0 +1,8 @@
+����� ���������� � �������� � �������� ������� ������ 
+ - �������� ����������������� ���������� �� ����� ���������� � ��������.
+ - ����������� �����������;
+ - - ������� ������;
+   - ������ � ��������� ������ "��������";
+   - ������ ������� "�����";
+ - ������ ������ ����� ��������� ������ � ����������� � ��������.
```

Comment:
Я нажал View и всё заработало на сайте


>Пятница,  8 апреля 2016, 18:40 +06:00 от Gleb Golovin <notifications@github.com>:
>
>In  Text.txt :
>> @@ -0,0 +1,8 @@
> +����� ���������� � �������� � �������� ������� ������ 
> + - �������� ����������������� ���������� �� ����� ���������� � ��������.
> + - ����������� �����������;
> + - - ������� ������;
> +   - ������ � ��������� ������ "��������";
> +   - ������ ������� "�����";
> + - ������ ������ ����� ��������� ������ � ����������� � ��������.
>windows-1251 чтоли? надо UTF-8
>проверять надо что отдаешь на ПР, в том числе открывать изменения тут в интерфейсе
>—
>You are receiving this because you authored the thread.
>Reply to this email directly or  view it on GitHub


С уважением,
Вова Кулогекун
aleks.kryp10@mail.ru